### PR TITLE
PHP agent release 9.19.0 (Release date: March 2, 2022)

### PR DESCRIPTION
**Pull request by misiekn**
Updates for the PHP agent v9.19.0:
* Add 'What's New' announcement
* Add 'PHP agent v9.19.0' release notes
* Update APM/PHP Agent/Compatibility and requirements
* Update APM/PHP Agent/Installation instructions

### CODE REVIEW

**Review by paperclypse — APPROVED** [review]
I've reviewed this and made some minor fixes. Looks good! Approved for the March 9 release date. @misiekn

**Comment by misiekn** [comment]
> I've reviewed this and made some minor fixes. Looks good! Approved for the March 9 release date. @misiekn

Thank you @paperclypse!

**Comment by barbnewrelic** [comment]
@jkinmonth @daynaslord @alexiskjones  The PHP release has been moved up to today, per @misiekn . My understanding is that at least one of you must approve the What's New post in this PR before it can go live. Can you please expedite? Thanks!

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+subject: PHP agent
+releaseDate: '2022-03-02'
+version: 9.19.0.309
```
Comment (daynaslord) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx:4]: Idk if we have a section for "version:"
If we do, keep this as is. If not, add "version 9.19....." to the title

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+downloadLink: 'https://download.newrelic.com/php_agent/archive/9.19.0.309'
```
Comment (daynaslord) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx:5]: Change to:
getStartedLink: '....'

File: src/content/whats-new/2022/03/php-agent.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+title: 'PHP agent now supports PHP version 8.1!!'
```
Comment (daynaslord) [inline src/content/whats-new/2022/03/php-agent.md:2]: Remove exclamation points

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+subject: PHP agent
```
Comment (daynaslord) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx:2]: Needs this format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'

File: src/content/docs/apm/agents/php-agent/installation/update-php-agent.mdx [hunk 5]
```diff
@@ -24,10 +24,10 @@ To update via **tar archive**, follow the [tar archive installation procedures](
```
Comment (daynaslord) [inline src/content/docs/apm/agents/php-agent/installation/update-php-agent.mdx:24]: Needs this format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'

File: src/content/docs/apm/agents/php-agent/installation/update-php-agent.mdx [hunk 6]
```diff
@@ -24,10 +24,10 @@ To update via **tar archive**, follow the [tar archive installation procedures](
 
 ## Use the `newrelic-php5` package [#newrelic-php5]
 
-The package name for the New Relic for PHP agent is `newrelic-php5`.
+The package name for New Relic's PHP agent is `newrelic-php5`.
```
Comment (daynaslord) [inline src/content/docs/apm/agents/php-agent/installation/update-php-agent.mdx:27]: What do folks have to do here? Install? Please confirm with Michal

File: src/content/docs/apm/agents/php-agent/installation/php-agent-installation-ubuntu-debian.mdx [hunk 7]
```diff
@@ -17,7 +17,7 @@ Our [PHP agent](/docs/agents/php-agent/getting-started/new-relic-php#configurati
```
Comment (daynaslord) [inline src/content/docs/apm/agents/php-agent/installation/php-agent-installation-ubuntu-debian.mdx:17]: Can this be combined with another post? If not, this needs this format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'

File: src/content/docs/apm/agents/php-agent/installation/php-agent-installation-tar-file.mdx [hunk 8]
```diff
@@ -23,7 +23,7 @@ Download the appropriate tar distribution file from **[download.newrelic.com/php
 * For Alpine Linux, download `newrelic-php5-X.X.X.X-linux-musl.tar.gz`.
```
Comment (daynaslord) [inline src/content/docs/apm/agents/php-agent/installation/php-agent-installation-tar-file.mdx:23]: Needs this format 

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'

File: src/content/docs/apm/agents/php-agent/installation/php-agent-installation-aws-linux-redhat-centos.mdx [hunk 9]
```diff
@@ -18,7 +18,7 @@ Our PHP agent auto-instruments your code so you can start monitoring application
```
Comment (daynaslord) [inline src/content/docs/apm/agents/php-agent/installation/php-agent-installation-aws-linux-redhat-centos.mdx:18]: Needs this format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'

File: src/content/docs/apm/agents/php-agent/installation/php-agent-installation-arm64.mdx [hunk 10]
```diff
@@ -34,7 +34,7 @@ ARM64 is only supported for Amazon Linux 2 and CentOS Linux 8 (including use wit
     sudo amazon-linux-extras install -y  php8.0
```
Comment (daynaslord) [inline src/content/docs/apm/agents/php-agent/installation/php-agent-installation-arm64.mdx:34]: Needs this format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'

File: src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx [hunk 11]
```diff
@@ -17,16 +17,20 @@ If you don't have one already, [create a New Relic account](https://newrelic.com
```
Comment (daynaslord) [inline src/content/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements.mdx:17]: Needs format

title: 'Insert a title here' 
summary: 'Insert a brief description of the feature' 
releaseDate: 'yyyy-mm-dd' 
learnMoreLink (optional): 'https://example.com' 
getStartedLink (optional): 'https://example.com'

**Review by daynaslord — CHANGES_REQUESTED** [review]
A few things:
- Formatting changes are needed (e.g. there should be a summary or sentence in the body that describes benefit/change/give description of features)
- There are 8 different posts here all re. PHP - Can some be combined? 

Thanks

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx (hunk 1 above)
Comment (misiekn) [reply src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx:4]: The subject and version are combined to create page's title. Please have a look at how release notes look like in [preview](https://docswebsitedevelop-misiekndocswebsitephpagentrele.gtsb.io/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309/).

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx (hunk 2 above)
Comment (misiekn) [reply src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx:5]: Using downloadLink is consistent with release notes for previous releases of the agent. Please take a look at previous two releases: [v9.17.1](https://github.com/newrelic/docs-website/blob/develop/src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9171301.mdx) and [v9.18.0](https://github.com/newrelic/docs-website/blob/develop/src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9181303.mdx). Please see on this screenshot to the right how it renders:
image

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx (hunk 4 above)
Comment (misiekn) [reply src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx:2]: I followed the format based on release notes for previous releases of the agent. Please take a look at previous two releases: [v9.17.1](https://github.com/newrelic/docs-website/blob/develop/src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9171301.mdx) and [v9.18.0](https://github.com/newrelic/docs-website/blob/develop/src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9181303.mdx).

**Comment by misiekn** [comment]
Hi @daynaslord. Thank you for taking the time to review this PR!

> * There are 8 different posts here all re. PHP - Can some be combined?

There's only one 'What's New' announcement post in this PR: [src/content/whats-new/2022/03/php-agent.md](https://github.com/newrelic/docs-website/pull/6380/files?file-filters%5B%5D=.md&show-viewed-files=true#diff-342388f0ab2bd444a766e7a9a56f7b209e3f99ef0dd2c69dd8019c3f1647ffd1) This PR additionally contains docs website updates required by the release of new version of one of New Relic's APM agents - PHP agent v9.19.0 specifically. This PR makes updates in following areas of docs website:

* Adds new 'What's New' announcement
* Adds new 'PHP agent v9.19.0' release notes 
* Updates existing documentation in PHP APM agent specific areas:
-- Update existing APM/PHP Agent/Compatibility and requirements
-- Update existing APM/PHP Agent/Installation instructions

I made updates suggested by you to the metadata of 'What's New' announcement. However I think that the metadata of release notes as well as existing PHP agent documentation pages should not be changed. If you think that release of new version of one of New Relic's APM agents, that adds long anticipated support for new version of PHP, shouldn't be announced in 'What's New', please let me know, and I'll remove 'What's New' announcement from this PR.

**Review by bradleycamacho — APPROVED** [review]

**Pull request merged**